osu.Game.Tests/Visual/Online/TestSceneChatOverlay.cs

```diff
@@ -207,6 +207,63 @@ public void TestChannelCloseButton()
             AddStep("Click normal close button", () => clickDrawable(((TestChannelTabItem)chatOverlay.TabMap[channel1]).CloseButton.Child));
             AddAssert("All channels closed", () => !channelManager.JoinedChannels.Any());
         }
+        
+        private void pressControlW()
+        {
+            AddStep("Press and hold Control", () => InputManager.PressKey(Key.ControlLeft));
+            AddStep("Press W", () => InputManager.Key(Key.W));
+            AddStep("Release Control", () => InputManager.ReleaseKey(Key.ControlLeft));
+        }
+        
+        [Test]
+        public void TestCtrlWShortcut()
+        {
+            AddStep("Join 2 channels", () => 
+            {
+                channelManager.JoinChannel(channel1);
+                channelManager.JoinChannel(channel2);
+            });
+
+            // Want to close channel 2
+            AddStep("Select channel 2", () => clickDrawable(chatOverlay.TabMap[channel2]));
+
+            pressControlW();
+            
+            // Channel 2 should be closed
+            AddAssert("channel 1 open", () => channelManager.JoinedChannels.Contains(channel1));
+            AddAssert("channel 2 closed", () => !channelManager.JoinedChannels.Contains(channel2));
+
+            // Want to close channel 1
+            AddStep("Select channel 1", () => clickDrawable(chatOverlay.TabMap[channel1]));
+
+            pressControlW();
+
+            // Channel 1 and channel 2 should be closed
+            AddAssert("channel 1 closed", () => !channelManager.JoinedChannels.Contains(channel1));
```

Comment:
I think this part can be combined into "All channels closed":
`AddAssert("All channels closed", () => !channelManager.JoinedChannels.Any());`